pkgs/development/python-modules/pymediainfo/default.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, fetchPypi, buildPythonPackage
+, libmediainfo
+, pytest, glibcLocales }:
+
+buildPythonPackage rec {
+  pname = "pymediainfo";
+  version = "2.2.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "160yyjyfg441h30pqfr6370hcch17j3s0x2f1l0r220xgy969mr1";
+  };
+
+  postPatch = ''
+    substituteInPlace pymediainfo/__init__.py \
+      --replace 'CDLL(library_file)' \
+                'CDLL("${libmediainfo}/lib/libmediainfo${stdenv.hostPlatform.extensions.sharedLibrary}")' \
+      --replace 'CDLL("libmediainfo.0.dylib")' \
+                'CDLL("${libmediainfo}/lib/libmediainfo.0${stdenv.hostPlatform.extensions.sharedLibrary}")' \
+      --replace 'CDLL("libmediainfo.dylib")' \
+                'CDLL("${libmediainfo}/lib/libmediainfo${stdenv.hostPlatform.extensions.sharedLibrary}")' \
+      --replace 'CDLL("libmediainfo.so.0")' \
+                'CDLL("${libmediainfo}/lib/libmediainfo${stdenv.hostPlatform.extensions.sharedLibrary}.0")'
```

Comment:
Is `.0` necessary?

Reply:
AFAIK this indicates the version number of the library which will be incremented on breaking changes. Thus I assume that this makes a difference.